ajax/libs/jquery.tablesorter/package.json

```diff
@@ -17,10 +17,6 @@
     "alphanumeric",
     "natural"
   ],
-  "licenses": [
-    "MIT",
-    "deprecated_GPL-2.0+"
```

Comment:
Please help revise here instead of adding new lines to minimize changes. Thanks.

Reply:
I do not get what is wrong I did???

Reply:
The content is correct for sure. Just hope to minimize the change. I can't see what the file content looks link now, so I can't judge on the issue you mentioned. I'll help do the simple change this time.